index.js

```diff
@@ -24,16 +24,13 @@ const ctx = canvas.getContext("2d"); //to acces the drawing context
 let randomMode = false;
 //For do the animation for each loop
 let reqAnimation;
-//To indicate if cycle is ON or OFF
-let started;
-let stoped = false;
 
 /* ==============================  BUILDING THE CANVAS GRID  ============================== */
 
 // Defining the canvas size, we can modify this values if we want to resize the grid
 const resolution = 10;
-const h = (canvas.width = 200);
-const w = (canvas.height = 200);
+const w = (canvas.width = 500);
```

Comment:
Should we remove the parenthesis?

Reply:
We can , I tried in my local branch and in the end it's the same result